com.w1hkj.flarq.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Flarq
+GenericName=Fast Light Automatic Repeat reQuest
+Comment=Fast Light Automatic Repeat reQuest
+Exec=flatpak run --command=flarq com.w1hkj.fldigi
+Icon=com.w1hkj.flarq.png
```

Comment:
Remove the `.png` suffix.

Reply:
Also, don't forget to change it to `com.w1hkj.fldigi.flarq` after you rename the desktop/icon files.